skgstat/MetricSpace.py

```diff
@@ -0,0 +1,120 @@
+import scipy.spatial
+import scipy.spatial.distance
+import scipy.sparse
+import numpy as np
+
+def sparse_dok_get(m, fill_value=np.NaN):
+    mm = np.full(m.shape, fill_value)
+    for (x, y), value in m.items():
+        mm[x,y] = value
+    return mm
```

Comment:
This function should be documented and tested. Especially if it is public. If it's not, we could add a underscore-prefix.
I, myself, don't get what it does, so a docstring would be nice to have anyway